nuxt.config.js

```diff
@@ -88,6 +95,7 @@ export default {
     sockets: [
       {
         default: true,
+        name: 'uitdatabank',
         url: 'https://sockets.uitdatabank.dev',
```

Comment:
Note that this URL is still hardcoded for now, due to a bug in `nuxt-socket-io`. See https://github.com/richardeschloss/nuxt-socket-io/issues/132